source/Grid/Grid.test.js

```diff
@@ -120,43 +120,49 @@ describe('Grid', () => {
   describe('shows and hides scrollbars based on rendered content', () => {
     const scrollbarSize = getScrollbarSize()
 
+    function renderTwice (markup) {
```

Comment:
What I originally suggested was to...
* Measure scrollbar size in [`componentWillMount`](https://facebook.github.io/react/docs/component-specs.html#mounting-componentwillmount) so that client-side content will only be measured once, right before rendering the first time.
* Check for an undefined scroll bar size in [`componentDidMount`](https://facebook.github.io/react/docs/component-specs.html#mounting-componentdidmount) so that server-side rendered content will be re-measured for scrollbar size at the first opportunity. In this second case, if scrollbar isn't measured until `componentDidMount`, then the `Grid` should `forceUpdate` (or `setState`) itself to ensure that it renders one more time with the adjusted scrollbar size.